opentracing-spring-web/src/main/java/io/opentracing/contrib/spring/web/interceptor/HandlerInterceptorSpanDecorator.java

```diff
@@ -71,6 +71,7 @@ public void onPreHandle(HttpServletRequest httpServletRequest, Object handler, B
             metaData = HandlerUtils.methodName(handler);
             if (metaData != null) {
                 logs.put(HandlerUtils.HANDLER_METHOD_NAME, metaData);
+                span.setOperationName(metaData);
```

Comment:
no log is not necessary.. What if `metaData` is null for some cases that operation name would be set to http verb and one would expect hadler method name. Could you please loot at docs if it can be null. If this is a reliable enough to use it as a default operation name.